include/framework/seadFramework.h

```diff
@@ -22,7 +22,7 @@ class LogicalFrameBuffer;
 class MethodTreeMgr;
 class TaskMgr;
 
-class Framework
+class Framework : hostio::Node
```

Comment:
```suggestion
class Framework : public hostio::Node
```
(unless there's a specific reason for making this private?)